Hazel/src/Hazel/Math/Math.h

```diff
@@ -3,7 +3,5 @@
 #include <glm/glm.hpp>
 
 namespace Hazel::Math {
-
-	bool DecomposeTransform(const glm::mat4& transform, glm::vec3& outTranslation, glm::vec3& outRotation, glm::vec3& outScale);
-
+	bool DecomposeTransform(const glm::mat4& transform, glm::vec3& translation, glm::vec3& rotation, glm::vec3& scale);
```

Comment:
I remember he wanted to prefix them with out to indicate they are the output... In any case, you should leave the new lines in there

```suggestion

	bool DecomposeTransform(const glm::mat4& transform, glm::vec3& translation, glm::vec3& rotation, glm::vec3& scale);

```

Reply:
Yeah, he wanted to do that. I just changed that to match the definition. In case, if he wants to change it, he can change it.